docs/annotate.rst

```diff
@@ -63,7 +62,27 @@ Step-by-step
 
    - The Key-Value Pairs allow you to add lab-book-like additional metadata for the Image. These Key-Value Pairs are also specifically searchable. See :doc:`search-omero`.
 
-#. It is also possible to add Comment, Attachment and Rating to selected Images in the right-hand pane in the ``General`` tab.
+#. For adding of File Attachments:
+
+   - Select one or more objects in the left-hand side tree, such as ``Dataset`` or ``Image``.
+   - Expand the ``Attachments`` harmonica in the right-hand pane. 
+   - Click the plus button.
+
+      |image1b|
+
+   - You can attach any type of file using this function. If you select a file from your local filesystem using the ``Browse`` button, the feature will upload that file to the OMERO.server and save it there. The content of ``.pdf``, ``CSV`` and plain text files is also searchable in OMERO.
+
+#. Remove a File Attachment.
+
+   - Find the File Attachment you have just added.
+
+   - Click on the ``minus`` sign to the right of it.
+
+      |image1c|
+
+   - The removal action just unlinks the File Attachment from the particual selected object(s). The File Attachment is not deleted from the server. If deletion is needed, click in the workflow above on the ``cross`` icon instead of the ``minus`` icon.
```

Comment:
```suggestion
   - The removal action just unlinks the File Attachment from the selected object(s). The File Attachment is not deleted from the server. If deletion is needed, click in the workflow above on the ``cross`` icon instead of the ``minus`` icon.
```